orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -1574,13 +1574,13 @@ TEST(GradientCheckerTest, SigmoidGrad) {
   UnaryOpGradientTest("Sigmoid");
 }
 
-void GradientCheckerSoftmaxGradHelper(bool is_log_softmax) {
+void GradientCheckerSoftmaxGradHelper(bool is_log_softmax, int version = 0) {
   TensorShape shape({3, 4, 5});
   float max_error;
   GradientChecker<float, float, float> gradient_checker;
 
   const std::string op = is_log_softmax ? "LogSoftmax" : "Softmax";
-  OpDef op_def{op};
+  OpDef op_def = version == 0 ? OpDef{op} : OpDef{op, kOnnxDomain, version};
```

Comment:
How about set the default value for version to 11? 

OpDef op_def = OpDef{op, kOnnxDomain, version};

Reply:
This is the existing UT. The default is set by the GradientChecker to 1 here
https://github.com/microsoft/onnxruntime/blob/b6fc9007f45f19119cd4bece7bce2a19d6d1f6a4/orttraining/orttraining/test/gradient/gradient_checker.cc#L300
Should we change it?

Reply:
Done